packages/gdrcopy.cfg

```diff
@@ -0,0 +1,17 @@
+### Commented entries have reasonable defaults.
+### Uncomment to edit them.
+# Source: <source package name; defaults to package name>
+Section: misc
+Priority: optional
+Homepage: https://github.com/NVIDIA/gdrcopy
+Standards-Version: @FULL_VERSION@
+
+Package: gdrcopy
+Version: @FULL_VERSION@
+Maintainer: Davide Rossetti <drossetti@nvidia.com>
```

Comment:
@pakmarkthub you should be mentioned here as Maintainer

Reply:
Quite reluctant to do but I have changed it.

By the way, is there a practice to put the team's name there instead. That would make it more correct.

Reply:
can we have 2 maintainers? either 2 lines or single line with 2 names separated somehow